### DISCUSSION TITLE
How to custom the display of HasMany field ?

### DISCUSSION THREAD

**Discussion by benjy8001** [moonshine-software/moonshine#1378] [Q&A]
Hi,

I try to customize the rendering of the HasMany field (for exemple display as Card instead of Table), so i created a custom IndexPage, with `itemsComponent`, but it's seems that is not the correct way to do it.

Is it possible to custom the display of HasMany field ? And how to do this ?

Thanks !

**Comment by lee-to** [comment]
Of course, you can implement it through changeRender method, but the best solution would be to write your own HasMany field with display in the form of cards (it would even be great if you add it as a package). It's not difficult, fields in MoonShine are very easy to make

You can also publish a page and output data from the relation via the CardsBuilder component + add a button with adding in the modal and this will also not be a difficult solution
